views/partials/truckProfile.jade

```diff
@@ -0,0 +1,55 @@
+#wrapper
+   #sidebar-wrapper.bg-color-mold
+      .container-fluid.make-space-for-logo.ignore-padding-right
+         div(data-ng-include="'/partials/shared/panelLeftMenu.jade'")
+         div.col-xs-12(style="padding-top: 30px;")
```

Comment:
Don't need to say div if you have a class: .col-xs-12 will be a div. This applies to every other div here that has a class